test/unit/latest_news_test.rb

```diff
@@ -0,0 +1,38 @@
+require 'test_helper'
+
+class LatestNewsTest < ActiveSupport::TestCase
+  setup do
+    Setting[:latest_news_rss_feed] = File.expand_path(File.dirname(__FILE__) + '/rss_feed.xml')
```

Comment:
Well, sure, it's indirectly testing the behavior but stubs will be more complicated.  I test the LatestNews service returns the right number of items, caches them for the specified period of time, handles exceptions, etc.  Having an RSS fixture seemed simpler...